xbmc/video/ContextMenus.cpp

```diff
@@ -67,7 +67,9 @@ bool CMarkWatched::IsVisible(const CFileItem& item) const
       return CUtil::IsTVRecording(item.GetPath());
   }
   else if (!item.HasVideoInfoTag())
-    return false;
+  {
+    return item.IsVideo() && !item.IsAudio(); // FIXME: For now we need to exclude audio items but this also excludes playlists like m3u, strm, xsp etc.
```

Comment:
Same "fix" is needed for "CMarkUnwatched', same source file.

Reply:
@ksooo : Thanks for the info. Will look into it when I have some time again. One thing I'm wondering about, since I don't know much about how the context menus are implemented atm: Do we really want this window logic in CMarkWatched/CMarkUnwatched or do we want this at a higher level? Why not simply only show all contextmenu items for video only for the video/pvr window? From what @DaveTBlake  is saying it I get that there is more stuff that needs the same attention... ?